src/Microsoft.DotNet.Build.Tasks/GenerateDepsJson.cs

```diff
@@ -29,6 +29,8 @@ public FileNameAssemblyPair(AssemblyName assemblyName, string fileName)
             }
         }
 
+        public bool GenerateLegacyDepsJson { get; set; }
```

Comment:
If we want to be backwards compatible then we need to reverse the default.

Reply:
you are right, will do.